app/controllers/api/v0/api_controller.rb

```diff
@@ -91,6 +92,11 @@ def add_missing_api_key_warning_header
         # rubocop:enable Layout/LineLength
       end
 
+      def api_usage_tracking
+        usage_tags = ["endpoint:#{request.path}", "user_id:#{@user&.id || 'unauthenticated'}"]
+        ForemStatsClient.increment("api.usage.v1", tags: usage_tags)
+      end
+
```

Comment:
Using an `after_action` hook based on a FeatureFlag check for v0 because its entrypoint + authentication is a bit more spread out, but essentially the same as v0 + unauthenticated possibility